neuralmonkey/vocabulary.py

```diff
@@ -352,6 +357,24 @@ def trunkate(self, size: int) -> None:
         for index, word in enumerate(self.index_to_word):
             self.word_to_index[word] = index
 
+    def truncate_by_min_freq(self, min_freq: int) -> None:
+        """Truncate the vocabulary by only keeping tokens with given minimum
+        frequency.
+
+        Arguments:
+            min_freq: The minimum frequency of included words.
+        """
+        if min_freq == 1:
```

Comment:
This one is not necessary since the `truncate_by_min_freq` function expects an integer.
The other case is dealt with, `from_dataset` now only calls `truncate_by_min_freq` when a value was given for `min_freq`